base/gfspvgmgr/virtual_group_manager.go

```diff
@@ -91,7 +91,17 @@ func (vgfm *virtualGroupFamilyManager) pickVirtualGroupFamily(filter *vgmgr.Pick
 	picker.freeStorageSizeWeightMap = make(map[uint32]float64)
 	for id, f := range vgfm.vgfIDToVgf {
 		if filter.Check(id) {
-			picker.addVirtualGroupFamily(f)
+			valid := true
+			for _, gvg := range f.GVGMap {
```

Comment:
Add a new family filter? bwt, can one bucket correspond to multiple families?

Reply:
filterByFaimlyID *vgmgr.PickVGFFilter, filterByGvgList  *vgmgr.PickVGFByGvgListFilter

f = family
if filterByFaimlyID.check(f.familyid) {
  continue
}

if filterByGvgList.check(f.gvgList) {
continue
}

if filterByFreeQuota.check() {
}


struct PickVGFByGvgListFilter interface {
     SPBlockList
}

Reply:
adjusted